src/open_inwoner/templates/pages/sitemap/sitemap.html

```diff
@@ -0,0 +1,60 @@
+{% extends "master.html" %}
+{% load i18n list_tags %}
+
+{% block content %}
+    <h1 class="utrecht-heading-1">{% trans "Links naar pagina's op het Open Inwoner Platform" %}</h1>
+
+    <div class="grid sitemap">
+        {# onderwerpen #}
+        <section class="sitemap__section">
+            <a class="utrecht-heading-2" href="{% url 'products:category_list' %}">{% trans "Onderwerpen" %}</a>
+            <ul class="sitemap__list">
+                {% for node in category_nodes %}
+                    {% include "pages/sitemap/category_item.html" with node=node %}
+                {% endfor %}
+            </ul>
+        </section>
+
+        {% if request.user.is_authenticated %}
+            <section class="sitemap__section">
+                {# profile pages heading #}
+                {% for page in cms_profile_pages|slice:":1" %}
+                    {% with link=page.url_name text=page.link_text %}
+                        <a class="utrecht-heading-2" href="{% url link %}">{% trans text %}</a>
+                    {% endwith %}
+                {% endfor %}
+                <ul class="sitemap__list">
+                    {# profile pages #}
+                    {% for page in cms_profile_pages|slice:"1:" %}
+                        {% with link=page.url_name text=page.link_text %}
+                            <li class="sitemap__list-item"><a href="{% url link %}">{% trans text %}</a></li>
+                        {% endwith %}
+                    {% endfor %}
+                    {# other CMS pages #}
+                    {% for page in cms_pages %}
+                        {% with link=page.url_name text=page.link_text %}
+                            <li class="sitemap__list-item"><a href="{% url link %}">{% trans text %}</a></li>
+                        {% endwith %}
+                    {% endfor %}
+                </ul>
+            </section>
+        {% endif %}
+
+        {# pages with information about OIP #}
+        <section class="sitemap__section">
+            <span class="utrecht-heading-2">{% trans "Platform" %}</span>
```

Comment:
I would actually move the Platform all the way up since that's most common?
That's the only thing that I'd like to put up as a "**request changes**" for this review - all the rest I can pick up myself after approve and merge.